pkg/satellite/satelliteauth/claims.go

```diff
@@ -0,0 +1,37 @@
+package satelliteauth
+
+import (
+	"encoding/json"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// Claims represents data signed by server and used for authentication
+type Claims struct {
+	ID         uuid.UUID `json:"id"`
+	Email      string    `json:"email,omitempty"`
+	Expiration time.Time `json:"expires,omitempty"`
+}
+
+// JSON returns json representation of Claims
```

Comment:
Usually these methods aren't the right way to handle these. Usually you want to use `json.NewEncoder` and `json.NewDecoder` to use less memory and be able to stream things.